composer.json

```diff
@@ -7,8 +7,18 @@
         "php": ">=5.4.0",
         "react/promise": "~2.0"
     },
+    "require-dev": {
+        "phpunit/phpunit": "~4.5"
```

Comment:
The closest I could find is this: https://github.com/reactphp/child-process/pull/5#discussion_r13724410

IMHO this is still worth discussing, however this is somewhat unrelated to your other changes. I'd vote to open a (meta) issue on reactphp/react so that we can discuss and apply this for *all* components.